regtests/README.md

```diff
@@ -21,31 +21,92 @@
 
 # End-to-end regression tests
 
-## Run Tests With Docker Compose
+Regression tests are either run in Docker, using docker-compose to orchestrate the tests, or
+locally.
+
+## Prerequisites
 
-Tests can be run with docker-compose by executing
+It is recommended to clean the `regtests/output` directory before running tests. This can be done by
+running:
 
-```bash
-docker compose up --build --exit-code-from regtest
+```shell
+rm -rf ./regtests/output && \
+  mkdir -p ./regtests/output && \
+  chmod -R 777 ./regtests/output
 ```
 
-This is the flow used in CI and should be done locally before pushing to github to ensure that no environmental
-factors contribute to the outcome of the tests.
+## Run Tests With Docker Compose
 
-## Run all tests
+Tests can be run with docker-compose using the provided `./regtests/docker-compose.yml` file.
 
-Polaris REST server must be running on localhost:8181 before running tests.
+In this setup, a Polaris container will be started using the `apache/polaris:latest` image. If the
+`apache/polaris:latest` image is not available locally, it will be pulled from the Polaris Docker
+Hub repository.
 
-Running test harness will automatically run idempotent setup script.
+_This means that, by default, the tests will be run against the latest published version of
+Polaris._ If you want to run the tests against a locally-built Polaris image, for example to
+validate changes before pushing to GitHub, _you must build the image first_. This can be done by
+running the following command from the root of the Polaris project:
```

Comment:
This seems extremely unintuitive. Is it too late to fix this?

Reply:
To be clear, we shouldn't block the doc change on this behavior -- but if we could fix this behavior that would be great.

Reply:
I think this paragraph is written in a very anxiety-provoking tone. It's not that unnatural for developers to first build their code, then start end-to-end tests. Let me rephrase this and I think it could sound a lot more intuitive.

Reply:
I personally find the propose change to be ok, but willing to see a alternative.